### PR TITLE
X25519 MTI

### PR DESCRIPTION
**Pull request by loganaden**
https://github.com/tlswg/tls13-spec/issues/1359 for discussion.

### CODE REVIEW

File: draft-ietf-tls-rfc8446bis.md
Comment (legna37) [inline draft-ietf-tls-rfc8446bis.md:1]: - `
[Uploading envvars_backend.txt…]()
`

Comment (legna37) [inline draft-ietf-tls-rfc8446bis.md:1]: - ``

**Comment by seanturner** [comment]
@ekr please close PR; see [mail](https://mailarchive.ietf.org/arch/msg/tls/T5l_cZD48t0fCWfUYV4b1l-hXbs/).

**Comment by ekr** [comment]
Closing per WG consensus.

**Pull request closed without merging**